link/src/test/java/com/stripe/android/link/ui/signup/SignUpViewModelTest.kt

```diff
@@ -220,18 +147,22 @@ internal class SignUpViewModelTest {
         val errorMessage = "Error message"
 
         val linkAccountManager = FakeLinkAccountManager()
+        val logger = FakeLogger()
         val viewModel = createViewModel(
             linkAccountManager = linkAccountManager,
             linkEventsReporter = object : SignUpLinkEventsReporter() {
                 override fun onSignupFailure(isInline: Boolean, error: Throwable) = Unit
-            }
+            },
+            logger = logger
         )
 
-        linkAccountManager.signUpResult = Result.failure(RuntimeException(errorMessage))
+        val exception = RuntimeException(errorMessage)
+        linkAccountManager.signUpResult = Result.failure(exception)
 
         viewModel.performValidSignup()
 
         assertThat(viewModel.contentState.errorMessage).isEqualTo(errorMessage.resolvableString)
+        assertThat(logger.errorLogs.last()).isEqualTo("SignUpViewModel Error: " to exception)
```

Comment:
do you expect there to be other errorLogs here? If not, it's probably worth verifying the size of the error logs + that this is the only entry